src/pages/ArticlePage.tsx

```diff
@@ -40,9 +56,32 @@ export const ArticlePage = ({ nonce }: { nonce?: string }) => {
               </>
             )}
           </GlobalStoryContainer>
+          <Wrapper>
+            {story && <ArticleBlock story={story} />}
+
+            <ArticleFooter>
+              {story?.content.article_footer.map((block, index) => {
+                const BlockComponent:
+                  | React.ComponentType<BaseBlockProps & any>
+                  | undefined = getBlockComponent(block.component)
+                if (!BlockComponent) {
+                  return null
+                }
```

Comment:
nit: this is becoming hard to read -- consider breaking out into separate component

Reply:
indeed! will take another stab at integrating the `react` SDK if it's not too much to change with all blocks 🙌